packages/babel-generator/src/token-map.ts

```diff
@@ -158,7 +158,8 @@ export class TokenMap {
     const indexes = [];
 
     for (const child of children) {
-      if (!child) continue;
+      if (child == null) continue;
+      if (child.start == null || child.end == null) continue;
```

Comment:
```suggestion
      if (child == null || child.start == null || child.end == null) continue;
```

Reply:
Heheh unfortunately typescript-eslint complains that `child == null || child.start == null` should be rewritten to `child?.start == null`, but I find `child?.start == null || child.end == null` to be terrible so this two-lines solution was the compromise I reached with the linter :P